plugins/BEdita/Core/src/ORM/Locator/TableLocator.php

```diff
@@ -37,10 +37,6 @@ class TableLocator extends CakeLocator
      */
     protected function _getClassName($alias, array $options = [])
     {
-        if (empty($options['className'])) {
-            $options['className'] = Inflector::camelize($alias);
-        }
-
```

Comment:
ok for removing this part. Consequently `$options['className']` is considered necessary parameter.
we could add in function doc a comment about that.
for instance:
* @param array $options Table options array. It must contain 'className'.